src/components/_tests_/Homepage.test.js

```diff
@@ -0,0 +1,19 @@
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import store from '../../redux/store/configureStore';
+import Homepage from '../homepage/Homepage';
+
+describe('Test Homepage component', () => {
+  it('should render the Homepage component correctly', () => {
+    const { container } = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Homepage />
+        </Provider>
+      </BrowserRouter>,
+    );
+
+    expect(container).toMatchSnapshot();
```

Comment:
> - [x] Some tests are failing currently. Let's fix this so that all the test passes 😄.
> 
> <img width="406" alt="Loom_b1f16fivwa" src="https://user-images.githubusercontent.com/81468675/153495251-772ba8e5-044d-4755-ab7e-6cb7f9f893e6.png">




- [ ] As required by the previous reviewer, Kindly make sure all tests passed before you request another review :point_down: 

![test](https://user-images.githubusercontent.com/57562869/153510285-de68d52f-b3a6-4e61-9a20-8c4011388ff7.png)


>

Reply:
@moise10r 
![image](https://user-images.githubusercontent.com/29071984/153511203-485075e4-73c6-4802-b0b0-723232c99b26.png)

Reply:
@moise10r 
As seen from your screenshot, It seems the tests are not running at all, which wasn't so for previous reviewers. If you could kindly have another look. Perhaps run `npm install` to confirm all dependencies are installed

Reply:
Hi @cwaku

Reply:
@moise10r Thank you.  I have limited submissions left, I really appreciate this.

Reply:
Hi @cwaku, Sorry for this, Seems to be an issue with my set-up, You can ask for another reviewer, and this will approve your project,

Reply:
@moise10r Oh okay thank you